sdk/setup.py

```diff
@@ -27,5 +27,5 @@
     classifiers=[
         "Programming Language :: Python :: 3",
     ],
-    python_requires=">=3.7,<=3.10",
```

Comment:
cc @kenxu95 @cw75 I think this removal is necessary to enable supports for `3.10`. Should we do `<=3.11` instead?

Reply:
I'd say just get rid of the upper restriction :)

Reply:
Also can you make a separate PR for the 310 stuff, just to keep this one from getting more cluttered?